src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,36 @@
+@import '/src/styles/blocks/variables';
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin: 0 0 24px;
+
+  &__total {
+    display: flex;
+  }
+
+  &__number-Of-Reviews {
+    color: rgba(6, 11, 53, 1);
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin: 0;
+  }
+
+  &__total--4 div:nth-child(-n + 4) {
+    background-image: url(/src/images/star-active.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &__star {
+    background-image: url(/src/images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+  }
+}
```

Comment:
The stars block should be independent and not know anything about its context, so the margin on line 7 should be removed. If you need to add a margin, it should be added to the parent block or element that uses the stars block.